website/docs/addon_aftereffects_artist.md

```diff
@@ -22,11 +22,12 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 
 ## Setup
 
-To install the extension, download, install [Anastasyi's Extension Manager](https://install.anastasiy.com/). Open Anastasyi's Extension Manager and select AfterEffects in menu. Then go to `{path to pype}hosts/aftereffects/api/extension.zxp`. 
+To install the extension, download, install [Anastasyi's Extension Manager](https://install.anastasiy.com/). Open Anastasyi's Extension Manager and select AfterEffects in menu. 
 
-(Current location will be most likely in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\core_X.X.X\ayon_core\hosts\aftereffects\api`.
-On Linux look in `~/.local/share/Ynput/AYON/addons`, on Mac `~/Library/Application Support/Ynput/AYON/addons`)
-This location is temporary until full separation of addons is implemented. Artist must start Tray at least once for Ayon server to populate this location.)
+Use extension.zxp downloaded with AfterEffects addon.
+Current location will be in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.
```

Comment:
I wonder if it'd be nice to add a TIP section that you can easily browse to the folder using `%LOCALAPPDATA%`
```suggestion
Current location will be in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\aftereffects_X.X.X\ayon_aftereffects\api`.
:::tip 
To quickly browse to the local appdata folder on your machine, use `%LOCALAPPDATA%` as path in the Windows explorer.
:::
```

Reply:
No. Just below it mentions it for Linux and mac?

Reply:
Thank you. added here https://github.com/ynput/ayon-documentation/pull/217/commits/57c13df08208942748e47cfae925082acc9c7e25